stdlib/out_channel.mli

```diff
@@ -164,3 +182,12 @@ val isatty : t -> bool
     [false] otherwise.
 
     @since 5.1 *)
+
+(** {1:examples Examples}
+   Writing a string to a file:
+    {[
+    # Out_channel.with_open_text "./example.txt"
+                    (fun oc -> Out_channel.ouptut_string oc "hello")
+    - : unit = ()
+    ]}
```

Comment:
Same remark here, `text` mode only if you are in for surprises. I would rather like to see something like:

```
Writing the contents of a file:
{[
let write_file file s = 
  Out_channel.with_open_bin file (fun oc -> Out_channel.output_string oc s))
]}
```

Reply:
Updated the example to use `with_open_bin` instead as suggested.